Tests/SwiftAlgorithmsTests/IsSortedTests.swift

```diff
@@ -0,0 +1,25 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Algorithms open source project
+//
+// Copyright (c) 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
+
+import XCTest
+import Algorithms
+
+final class IsSortedTests: XCTestCase {
```

Comment:
It would be good to test an empty collection, to check sorted collections that includes repeats (e.g. `[1, 1, 1, 2, 3]`), and also to verify that `isSorted` returns `false` for unsorted collections.